src/options.ts

```diff
@@ -199,10 +199,13 @@ function getVariableOptions() {
 function addTimeZone(builder: PanelOptionsEditorBuilder<ClockOptions>) {
   const category = ['Timezone'];
 
-  const timezones = getTimeZoneNames().map((n) => {
-    return { label: n, value: n };
-  });
-  timezones.unshift({ label: 'Default', value: '' });
+  const timezones = [
+    { label: 'Default (Browser)', value: '' },
+    { label: 'Same as Dashboard', value: 'dashboard' },
+    ...getTimeZoneNames().map((n) => {
+      return { label: n, value: n };
+    }),
+  ];
```

Comment:
What do you think about changing the UI for this a bit? For me it would make sense to have two options:

1. Same as dashboard
2. Custom for panel

When a user selects the first one we use the same as the dashboard is configured to use. When the user selects the other one we ask them for the time zone that they want to use.

When it comes to the options of that selector I think we should try to use the same list as we are using in the TimePicker.

So basically:

- Default
- Browser Time
- UTC
- ...time zones

Default can be configured on two levels. Either on the organisational level or for the user account.

Browser Time is what ever timezone your browser is set to.

Reply:
@mckn That's a good option but It'll probably require some migration from one version of the panel to the other since we currently store the timezone as a single field.

I also like the idea of keeping a familiar UI for people upgrading the panel. 

Regarding the "Default" in the second list. where can I get it from? In the props we only get two timezones: The one that comes from the dashboard and the one that the user could have selected from the selector.

Reply:
> @mckn That's a good option but It'll probably require some migration from one version of the panel to the other since we currently store the timezone as a single field.
> 
Yeah, I gave this a bit of a more thought and you are correct. Probably not worth changing too much in the UI. Stick with what you have!

> I also like the idea of keeping a familiar UI for people upgrading the panel.
> 
> Regarding the "Default" in the second list. where can I get it from? In the props we only get two timezones: The one that comes from the dashboard and the one that the user could have selected from the selector.

Hmm, if you call this function, without any options, `getTimeZone()`  from `grafana/data` you will get the currently selected default time zone.

Reply:
https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/datetime/common.ts#L59C14-L59C25

Reply:
@mckn if you are ok with not changing the UI. Should we merge this PR as is? Or should we add an option to "use default timezone"?

The whole "default" thing gets now more complicated. We have now the Browser Time. The Dashboard Time. The User-defined time (in the panel) and the grafana default time.

Reply:
I think we can keep the UI but I think we should do one of the following:

- Remove the `Default (Browser)` since the default can be an other timezone than `browser` so that label is not correct.
- Rename `Default (Browser)` to `Default` and add another value called `Browser Time`. This correlates with the options in the Time Picker.

Reply:
@mckn In this specific case the "default" in this plugin is indeed the browser. so `Default (browser)` is the correct one.

There's currently no logic getting the `Default` as the one coming from the grafana configuration. The plugin had always used the word "Default" to refer to the browser time, that's why I added the clarification in this PR.

Reply:
Ah, ok ok - then I would suggest to change the name from `Default` to `Browser Time` to align it with the time picker. It would make it much clearer and consistent.

Reply:
This is a perfect scenario where @sympatheticmoose usually have great input so I'm adding him here.

Reply:
Sorry this has taken so long to get to. 

Based on the above I would be tempted to rename to `Browser Time` so its consistent with the timezone picker in the dashboard. Hadn't realised that Default was actually "a thing" and one that we don't support currently (nor am I convinced that adding support for it is urgent). 

The middle ground would be `Browser Time (Default)` but if we then want to extend with support for `Default` at a later stage, this becomes more confusing.

Reply:
@sympatheticmoose @mckn I changed it to `Browser Time` that makes mores sense. No migrations required for this case since is just a label change.

Reply:
Amazing! Great work on this